requirements/development.txt

```diff
@@ -1,4 +1,4 @@
-black
+git+https://github.com/psf/black
```

Comment:
I don't think this is right, as a specific version would be preferred. In any event, we can discuss this in another pull request dealing only with updating black/pre-commit.

Reply:
Hi Graham, I didn't intend to push any of these changes for black and the version. I only made these changes locally for myself because I ran into [this error ](https://issueantenna.com/repo/psf/black/issues/2964) while trying to commit and used the solution they suggested in that thread. I didn't add the precommit yaml file or development.txt to my commit/pull request. Not sure how they got pushed. 

I fixed all the issues with tests on my computer before I submitted the pull request, and didn't have any tests failing at that point. How should I fix these issues? Thanks!

Reply:
I'm off to bed now, but I'll take a closer look tomorrow and see if I can figure out the failing tests.